### PR TITLE
Add variety in taxon rank

### LINKED ISSUE #3246: Support Variety taxon rank
Currently we can't upload taxa with variety rank, e.g. https://www.gbif.org/species/6285615. 

Add Variety taxon rank to the system, and update the uploaders.

### PR DESCRIPTION
**Pull request by Fanevanjanahary**
fixes #3246

### CODE REVIEW

File: bims/tasks/taxa_upload.py [hunk 1]
```diff
@@ -7,7 +7,8 @@
     TAXON,
     SCIENTIFIC_NAME,
     ORIGIN,
-    ENDEMISM
+    ENDEMISM,
+    VARIETY
```
Comment (dimasciput) [inline bims/tasks/taxa_upload.py:11]: @Fanevanjanahary please don't add the variable here, VARIETY is optional header.

**Review by dimasciput — COMMENTED** [review]
@Fanevanjanahary did you manage to upload the sample data? Can you screenshot the uploaded results in the admin page here.

**Comment by Fanevanjanahary** [comment]
Yes @dimasciput 

image

**Comment by dimasciput** [comment]
Thanks LGTM

**Pull request merged**